docs/software/media.md

```diff
@@ -47,4 +43,818 @@
+
+![DietPi media server software Airsonic](../assets/images/dietpi-software-media-airsonic.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    - URL = `http://<your.IP>:8080/airsonic`
+    - Username = `admin`
+    - Password = `admin`
+
+=== "First run setup"
+
+    Once connected to the web interface, we need to configure the Music directory:
+
+    - Click the `Settings` button
+    - Under `Media Folders`, replace the current music folder entry with `/mnt/dietpi_userdata/Music`, then select `Save`
+    - Select `Scan media folders now` to update the library
+
+=== "Transfer/add music"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by Mopidy:
+
+    - folder = `/mnt/dietpi_userdata/Music`  
+
+## Subsonic - Feature rich media streaming server with web interface
+
+Feature rich media streaming server with a web interface. Coded in Java.
+
+![DietPi media server software Subsonic](../assets/images/dietpi-software-media-subsonic.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    - URL = `http://<your.IP>:4040`
+    - Username = `admin`
+    - Password = `admin`
+
+=== "Transfer/add music"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by Mopidy:
+
+    - folder = `/mnt/dietpi_userdata/Music`  
+
+=== "Jetty http 404 error"
+
+    This is a known issue with Subsonic and is out of my control.  
+    If you experience this error while accessing the Subsonic web page, you can resolve the issue by running the following:
+
+    ```sh
+    systemctl stop subsonic
+    rm -R /var/subsonic/jetty
+    systemctl restart subsonic
+    ```
+
+=== "GitHub Contributions"
+
+    Many thanks to 'thechildofroth' for his testing and fixes with the Subsonic installation.
+
+## Logitech Media Server - aka LMS, Squeezebox Server
+
+Logitech Media Server is the server software that enables web interface control of:
+
+- Software audio players: SqueezeLite
+- Hardware audio players from Logitech: Squeezebox, SlimDevices
+
+Remark: If you do not own a hardware player, you can turn your DietPi system into a Squeezebox audio capable player, by selecting SqueezeLite for installation in `dietpi-software`.
+
+![DietPi media server software Logitech Media Server](../assets/images/dietpi-software-media-squeezebox.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    - URL = `http://<your.IP>:9000`
+
+=== "Transfer/add music"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by Mopidy:
+
+    - folder = `/mnt/dietpi_userdata/Music`  
+
+=== "Change SqueezeLite command line options"
+
+    - Run `dietpi-services`
+    - Select `squeezelite`
+    - Select `Edit`
+    - Unset and re-set the `ExecStart` entry:
+      :
```

Comment:
Should have been an empty line here?
2nd case below SqueezeLite.

Hmm, do we want to keep the info doubled or remove SqueezeLite related info from LMS docs? We could also change `Software audio players: SqueezeLite` to be a ling to below header:
```
- Software audio players: [SqueezeLite](#squeezelite-audio-player-for-logitech-media-server)
```

Reply:
An empty line (in combination with deleting the ':') would lead to a code block which is shifted more left. With this combination it looks better.

For now, I would keep the two sections to have all information at the place without jumping in the doc.  
Although the text is doubled and in case of changes inconsistencies may occur...